test-cypress/fixtures/example.json

```diff
@@ -0,0 +1,4 @@
+{
+  "username": "sava",
+  "password": "123"
```

Comment:
This properties are not used, we can remove the content of the file or the file itself.